projects/plugins/jetpack/modules/stats.php

```diff
@@ -562,130 +571,135 @@ function jetpack_admin_ui_stats_report_page_wrapper()  {
  */
 function stats_reports_page( $main_chart_only = false ) {
 
-	if ( isset( $_GET['dashboard'] ) ) {
+	if ( isset( $_GET['dashboard'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 		return stats_dashboard_widget_content();
 	}
 
 	$blog_id   = stats_get_option( 'blog_id' );
 	$stats_url = Redirect::get_url( 'calypso-stats' );
 
-	$jetpack_admin_url = admin_url() . 'admin.php?page=jetpack';
-
-	if ( ! $main_chart_only && ! isset( $_GET['noheader'] ) && empty( $_GET['nojs'] ) && empty( $_COOKIE['stnojs'] ) ) {
+	if ( ! $main_chart_only && ! isset( $_GET['noheader'] ) && empty( $_GET['nojs'] ) && empty( $_COOKIE['stnojs'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 		$nojs_url = add_query_arg( 'nojs', '1' );
-		$http = is_ssl() ? 'https' : 'http';
+		$http     = is_ssl() ? 'https' : 'http';
 		// Loading message. No JS fallback message.
-?>
+		?>
 
 	<div id="jp-stats-wrap">
 		<div class="wrap">
 			<h2><?php esc_html_e( 'Site Stats', 'jetpack' ); ?>
 			<?php
-				if ( current_user_can( 'jetpack_manage_modules' ) ) :
-					$i18n_headers = jetpack_get_module_i18n( 'stats' );
-			?>
+			if ( current_user_can( 'jetpack_manage_modules' ) ) :
+				$i18n_headers = jetpack_get_module_i18n( 'stats' );
+				?>
 				<a
 					style="font-size:13px;"
 					href="<?php echo esc_url( admin_url( 'admin.php?page=jetpack#/settings?term=' . rawurlencode( $i18n_headers['name'] ) ) ); ?>"
 				>
-					<?php esc_html_e( 'Configure', 'jetpack' ); ?>
+				<?php esc_html_e( 'Configure', 'jetpack' ); ?>
 				</a>
-			<?php
+				<?php
 				endif;
 			?>
 			</h2>
 		</div>
 		<div id="stats-loading-wrap" class="wrap">
-		<p class="hide-if-no-js"><img width="32" height="32" alt="<?php esc_attr_e( 'Loading&hellip;', 'jetpack' ); ?>" src="<?php
-				echo esc_url(
-					/**
-					 * Sets external resource URL.
-					 *
-					 * @module stats
-					 *
-					 * @since 1.4.0
-					 *
-					 * @param string $args URL of external resource.
-					 */
-					apply_filters( 'jetpack_static_url', "{$http}://en.wordpress.com/i/loading/loading-64.gif" )
-				); ?>" /></p>
+		<p class="hide-if-no-js"><img width="32" height="32" alt="<?php esc_attr_e( 'Loading&hellip;', 'jetpack' ); ?>" src="
+																					<?php
```

Comment:
That's an odd one :)

I wonder if we could make this a variable declared earlier, to avoid this odd indentation?

Reply:
Sure, if you want to kill fun. Updated!